admin_manual/installation/deployment_recommendations.rst

```diff
@@ -377,6 +377,19 @@ You may also encounter database errors, such as this one:
 The issue, `identified by Michael Roth`_, is caused when MariaDB Galera cluster sends write requests to all servers in the cluster; `here is a detailed explanation`_.
 The solution is to send all write requests to a single server, instead of all of them.
 
+---------
+
+Set the wsrep_sync_wait parameter to 1 on all nodes of your Galera Cluster. It is disabled per default.
+
+What this parameter actually does:
+
+When you enable this parameter, the node triggers causality checks in response to certain types of queries. During the check, the node blocks new queries while the database server catches up with all updates made in the cluster to the point where the check was begun. Once it reaches this point, the node executes the original query. source http://galeracluster.com/documentation-webpages/mysqlwsrepoptions.html
```

Comment:
It'd be ideal if http://galeracluster.com/documentation-webpages/mysqlwsrepoptions.html could be linked into the text, instead of being included as separate text.

Reply:
I don't know how to do it.